config/controller.go

```diff
@@ -47,7 +47,8 @@ func parseConfigMap(cm *apiv1.ConfigMap, config *Config) error {
 			}
 		}
 	}
-	err := yaml.UnmarshalStrict([]byte(rawConfig), config)
+	// err := yaml.UnmarshalStrict([]byte(rawConfig), config)
```

Comment:
revert

Reply:
Yes, this is on the Todo before merging, this was set for our testing.